src/python/pants/option/config.py

```diff
@@ -105,6 +106,25 @@ def _meta_load(
                 }
                 config_values = _TomlValues(toml_values)
             else:
+                script_instructions = (
+                    "curl -L -o migrate_to_toml_config.py 'https://git.io/Jv02R' && chmod +x "
+                    f"migrate_to_toml_config.py && ./migrate_to_toml_config.py {config_path}"
+                )
+                warn_or_error(
+                    removal_version="1.28.0.dev0",
+                    deprecation_start_version="1.26.0.dev1",
```

Comment:
This line is redundant, but I put it here to invite the conversation if we want to start the deprecation in 1.26.x or 1.27.x.
 
I advocate starting in 1.26.x because we expect it to take no more than 5-20 minutes for users to migrate (depending on the # lines of config) and this earlier start will allow us to simplify our config code sooner.

But, I acknowledge that 1.25.x had too many big breaking changes, and this is another big breaking change (although easier to address than deprecating `globs()`), so I understand if we want the 1.25.x deprecations to be removed in 1.27.x before making this change.